src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -354,11 +349,11 @@ bool AdaptDispatch::_SetRgbColorsHelper(const std::basic_string_view<DispatchTyp
 {
     bool success = false;
     optionsConsumed = 1;
-    if (options.size() >= 2 && s_IsRgbColorOption(options.at(0)))
+    if (options.size() >= 2 && _isRgbColorOption(options.at(0)))

```

Comment:
this could be a `til::at` because of the short-circuit before the `&&`

Reply:
ok